middleware/logger.go

```diff
@@ -203,14 +212,17 @@ func logger(config LoggerConfig) fiber.Handler {
 
 	var tmpl loggerTemplate
 	tmpl.new(config.Format, "${", "}")
-
-	timestamp := time.Now().Format(config.TimeFormat)
+	var location, err = time.LoadLocation(config.TimeZone)
+	if err != nil {
```

Comment:
I have my doubts about this integration.
It would be nice to have `Local` as default timezone and maybe as fallback.
@gofiber/maintainers, let me know what you think.
Thanks for your contribution! 🎉

Reply:
Agreed. `Local` is better.

Reply:
Yes, `Local` is already defined in `LoggerConfigDefault`.

https://github.com/gofiber/fiber/blob/1fa775fa29aeab066823a8914ba83c84747a0e6a/middleware/logger.go#L137-L143

Reply:
What if `LoggerConfig.TimeZone` is empty string?

You should set a default value after:
https://github.com/gofiber/fiber/blob/1fa775fa29aeab066823a8914ba83c84747a0e6a/middleware/logger.go#L198

Reply:
> What if `LoggerConfig.TimeZone` is empty string?
> 
> You should set a default value after:
> https://github.com/gofiber/fiber/blob/1fa775fa29aeab066823a8914ba83c84747a0e6a/middleware/logger.go#L198-199

1. If the name is "", `LoadLocation` returns “UTC”.
2. This is an oversight, I will solve it, sorry.

Reply:
Now there is a problem. The empty string in the Golang time package should be "UTC". But our default value is empty will conflict with it, do we want to change it to "Local"'?

Reply:
```go
if config.TimeZone == "" {
	config.TimeZone = LoggerConfigDefault.TimeZone
}
```